Header header
int32 robot_id
int32 state
int32 first_location
int32 second_location
bool is_the_last
string area
